internals/secrethub/service.go

```diff
@@ -21,4 +21,5 @@ func (cmd *ServiceCommand) Register(r Registerer) {
 	clause := r.Command("service", "Manage service accounts.")
 	NewServiceDeployCommand(cmd.io).Register(clause)
```

Comment:
Not something your PR changes, but let's order the commands logically: `init, ls, deploy`.

Reply:
Looked into it and it isn't a clear cut solution to switch the order around. We'll pick this up later.